models/mixins/relations/user.coffee

```diff
@@ -21,7 +21,7 @@ module.exports =
       options.data = @pick('email')
       fetch.call account, options
 
-    authentications = new Backbone.Collection
+    authentications = new Backbone.Collection @get 'authentications'
     authentications.url = "#{API_URL}/api/v1/me/authentications"
```

Comment:
It's been a _looong_ time for me, but is this any different than just calling `fetch` on the collection object after setting its URL? Alternatively, can we skip setting the collection's URL given that we're constructing the object with initial data?